linebot/flex.go

```diff
@@ -19,6 +19,14 @@ import "encoding/json"
 // FlexContainerType type
 type FlexContainerType string
 
+// FlexValueType type
+type FlexValueType *int
+
+// FlexValue is helper function for using FlexValueType values
+func FlexValue(v int) FlexValueType {
```

Comment:
I think that the function name better be `FlexInt`.

Reply:
Thanks, I renamed it.